test/dryrun/utils.go

```diff
@@ -16,54 +16,71 @@ import (
 	"open-cluster-management.io/config-policy-controller/pkg/dryrun"
 )
 
-func Run(testFiles embed.FS, filePath string) func(t *testing.T) {
+func Run(testFiles embed.FS) func(t *testing.T) {
 	return func(t *testing.T) {
 		t.Helper()
 
-		scenarioFiles, err := testFiles.ReadDir(filePath)
-		if err != nil {
-			t.Fatal(err)
-		}
-
 		d := dryrun.DryRunner{}
 		cmd := d.GetCmd()
 		testout := bytes.Buffer{}
+		wanted := []byte{}
+		wantedFile := ""
 
 		cmd.SetOut(&testout)
 
 		args := []string{"--no-colors"}
 
-		for _, f := range scenarioFiles {
-			if strings.HasPrefix(f.Name(), "input") {
-				args = append(args, path.Join(filePath, f.Name()))
+		err := fs.WalkDir(testFiles, ".", func(path string, file fs.DirEntry, err error) error {
```

Comment:
This is the portion that allows the function to not need to know the path argument.